pyrelational/model_managers/abstract_model_manager.py

```diff
@@ -9,16 +9,18 @@
 
 
 class ModelManager(ABC, Generic[ModelType, E]):
+    """
+    Abstract class used to wrap models to interact with the ActiveLearningStrategy modules.
```

Comment:
I think this needs to be updated to `ActiveLearningStrategy` -> `Strategy`

Reply:
thanks, I have changed it now :)